July Release Notes for public docs (#2060)
spryker/spryker-docs: docs/scos/user/intro-to-spryker/docs-release-notes.md

title: Docs release notes
description: Spryker docs release notes
template: concept-topic-template
last_updated: Aug 2, 2023
---
## July 2023

In July 2023, we have added and updated the following pages:

### New pages
- [Oryx: Color system](/docs/scos/dev/front-end-development/202212.0/oryx/styling/oryx-color-system.html): Documentation on colors in Oryx.
- [Best practises: Jenkins stability](/docs/cloud/dev/spryker-cloud-commerce-os/best-practices/best-practises-jenkins-stability.html): Best practices for maintaining Jenkins stability.
- [HowTo: Reduce Jenkins execution by up to 80% without P&S and Data importers refactoring](/docs/scos/dev/tutorials-and-howtos/howtos/howto-reduce-jenkins-execution-costs-without-refactoring.html): Save Jenkins-related costs or speed up background jobs processing by implementing a single custom Worker for all stores.
- [Release notes 202307.0](/docs/scos/user/intro-to-spryker/releases/release-notes/release-notes-202307.0/release-notes-202307.0.html): Release notes for the Spryker SCOS release 202307.0

> - [Install the Product Rating and Reviews Glue API](/docs/pbc/all/ratings-reviews/202304.0/install-and-upgrade/install-the-product-rating-and-reviews-glue-api.html): This document describes how to integrate the [Product Raiting and Reviews](/docs/pbc/all/ratings-reviews/{{site.version}}/ratings-and-reviews.html) Glue API feature into a Spryker project.

Author: ```suggestion
- [Install the Product Rating and Reviews Glue API](/docs/pbc/all/ratings-reviews/202307.0/install-and-upgrade/install-the-product-rating-and-reviews-glue-api.html): This document describes how to integrate the [Product Rating and Reviews](/docs/pbc/all/ratings-reviews/{{site.version}}/ratings-and-reviews.html) Glue API feature into a Spryker project.
```

description: Spryker docs release notes

> template: concept-topic-template

Reviewer: ```suggestion
template: concept-topic-template
last_updated: Aug 2, 2023
```

---
## July 2023

In [July 2023, we have added and updated the following pages:

### New pages
- [Oryx: Color system](/docs/scos/dev/front-end-development/202212.0/oryx/styling/oryx-color-system.html): Documentation on colours in Oryx.
- [Best practises: Jenkins stability](/docs/cloud/dev/spryker-cloud-commerce-os/best-practices/best-practises-jenkins-stability.html): Best practices for maintaining Jenkins stability.

> - [HowTo: Reduce Jenkins execution by up to 80% without P&S and Data importers refactoring](/docs/scos/dev/tutorials-and-howtos/howtos/howto-reduce-jenkins-execution-costs-without-refactoring.html): Save Jenkins related costs or speed up background jobs processing by implementing a single custom Worker for all stores.

Reviewer: ```suggestion
- [HowTo: Reduce Jenkins execution by up to 80% without P&S and Data importers refactoring](/docs/scos/dev/tutorials-and-howtos/howtos/howto-reduce-jenkins-execution-costs-without-refactoring.html): Save Jenkins-related costs or speed up background jobs processing by implementing a single custom Worker for all stores.
```

- [Release notes 202307.0](/docs/scos/user/intro-to-spryker/releases/release-notes/release-notes-202307.0/release-notes-202307.0.html): Release notes for the Spryker SCOS release 202307.0

> - [Install the Product Rating and Reviews Glue API](/docs/pbc/all/ratings-reviews/202304.0/install-and-upgrade/install-the-product-rating-and-reviews-glue-api.html): This document describes how to integrate the [Product Raiting and Reviews](/docs/pbc/all/ratings-reviews/{{page.version}}/ratings-and-reviews.html) Glue API feature into a Spryker project.

Reviewer: Fix the "Install the Product Rating and Reviews Glue API" link. It's broken

> - [Install the Product Rating and Reviews Glue API](/docs/pbc/all/ratings-reviews/202304.0/install-and-upgrade/install-the-product-rating-and-reviews-glue-api.html): This document describes how to integrate the [Product Raiting and Reviews](/docs/pbc/all/ratings-reviews/{{page.version}}/ratings-and-reviews.html) Glue API feature into a Spryker project.

Reviewer: Use `site.version` on the unversioned pages.
```suggestion
- [Install the Product Rating and Reviews Glue API](/docs/pbc/all/ratings-reviews/202304.0/install-and-upgrade/install-the-product-rating-and-reviews-glue-api.html): This document describes how to integrate the [Product Raiting and Reviews](/docs/pbc/all/ratings-reviews/{{site.version}}/ratings-and-reviews.html) Glue API feature into a Spryker project.
```

- [Install the Spryker Core feature](/docs/pbc/all/miscellaneous/202307.0/install-and-upgrade/install-features/install-the-spryker-core-feature.html): Updated code sample.
- [Install Docker prerequisites on Linux](/docs/scos/dev/set-up-spryker-locally/install-spryker/install-docker-prerequisites/install-docker-prerequisites-on-linux.html): Learn about the steps you need to take before you can start working with Spryker in Docker on Linux.
- [Payment Service Provider](/docs/pbc/all/payment-service-provider/202212.0/payment-service-provider.html): Different payment methods for your shop.
- [System requirements](/docs/scos/dev/system-requirements/202304.0/system-requirements.html): This document provides the configuration that a system must have in order for the Spryker project to run smoothly and efficiently.
- [Supported browsers](/docs/scos/user/intro-to-spryker/supported-browsers.html): This document lists browsers supported by Spryker Cloud Commerce OS.
- [Install the shipment feature](/docs/pbc/all/carrier-management/202400.0/unified-commerce/install-and-upgrade/install-the-shipment-feature.html)

> - [App Composition Platform installation](/docs/acp/user/app-composition-platform-installation.html)

Reviewer: add description

Author: Guide said it doesn't need a description if the name is enough

- [App manifest](/docs/acp/user/app-manifest.html): App Manifest is a set of JSON files that contain all the necessary information to register an application in the Application Tenant Registry Service and to display information about the application in the Application Catalog.
- [Development strategies](/docs/scos/dev/back-end-development/extend-spryker/development-strategies.html): Development strategies that you can use when building a Spryker project.

> - [Project development guidelines](/docs/scos/dev/guidelines/project-development-guidelines.html): This article describes the strategies a project team can take while building a Spryker-based project.

Reviewer: style guide
```suggestion
- [Project development guidelines](/docs/scos/dev/guidelines/project-development-guidelines.html): This document describes the strategies a project team can take while building a Spryker-based project.
```

> - [Handling security issues](/docs/scos/user/intro-to-spryker/support/handling-security-issues.html): Use this article to learn how to report a security issue and to understand how we handle these reports.     

Reviewer: ```suggestion
- [Handling security issues](/docs/scos/user/intro-to-spryker/support/handling-security-issues.html): Use this document to learn how to report a security issue and to understand how we handle these reports.     
```

description: Spryker docs release notes
template: concept-topic-template
---
## July 2023

> In [July 2023, we have added and updated the following pages:

Reviewer: ```suggestion
In July 2023, we have added and updated the following pages:
```

- [Development strategies](/docs/scos/dev/back-end-development/extend-spryker/development-strategies.html): Development strategies that you can use when building a Spryker project.
- [Project development guidelines](/docs/scos/dev/guidelines/project-development-guidelines.html): This article describes the strategies a project team can take while building a Spryker-based project.
- [Handling security issues](/docs/scos/user/intro-to-spryker/support/handling-security-issues.html): Use this article to learn how to report a security issue and to understand how we handle these reports.     
- [Install the Measurement Units feature](/docs/pbc/all/product-information-management/202307.0/base-shop/install-and-upgrade/install-features/install-the-measurement-units-feature.html): The Measurement Units feature allows defining specific units of measure for products. The guide describes how to integrate the feature into your project.

> - [Install the Service Points feature](/docs/pbc/all/service-points/202400.0/unified-commerce/install-the-service-points-feature.html): Learn how to integrate the Service Points feature into your project.

Author: ```suggestion
```

> For more details on these and other updates to the Spryker docs in July 2023, refer to the [docs release notes page on GitHub](https://github.com/spryker/spryker-docs/releases)

Reviewer: Style guide
```suggestion
For more details about these and other updates to the Spryker docs in July 2023, refer to the [docs release notes page on GitHub](https://github.com/spryker/spryker-docs/releases)
```
